doc/classes/Dictionary.xml

```diff
@@ -200,123 +146,140 @@
 			<description>
-				Returns the first key whose associated value is equal to [param value], or [code]null[/code] if no such value is found.
-				[b]Note:[/b] [code]null[/code] is also a valid key. If you have it in your [Dictionary], the [method find_key] method can give misleading results.
+				Finds and returns the first key whose associated value is equal to [param value], or [code]null[/code] if it is not found.
+				[b]Note:[/b] [code]null[/code] is also a valid key. If inside the dictionary, [method find_key] may give misleading results.
 			</description>
 		</method>
 		<method name="get" qualifiers="const">
 			<return type="Variant" />
 			<param index="0" name="key" type="Variant" />
 			<param index="1" name="default" type="Variant" default="null" />
 			<description>
-				Returns the current value for the specified key in the [Dictionary]. If the key does not exist, the method returns the value of the optional default argument, or [code]null[/code] if it is omitted.
+				Returns the corresponding value for the given [param key] in the dictionary. If the [param key] does not exist, returns [param default], or [code]null[/code] if the parameter is omitted.
 			</description>
 		</method>
 		<method name="has" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="key" type="Variant" />
 			<description>
-				Returns [code]true[/code] if the dictionary has a given key.
-				[b]Note:[/b] This is equivalent to using the [code]in[/code] operator as follows:
+				Returns [code]true[/code] if the dictionary contains an entry with the given [param key].
 				[codeblocks]
 				[gdscript]
-				# Will evaluate to `true`.
-				if "godot" in {"godot": "engine"}:
-				    pass
+				var my_dict = {
+				    "Godot" : 4,
+				    210 : null,
+				}
+				
+				print(my_dict.has("Godot")) # Prints true
+				print(my_dict.has(210))     # Prints true
+				print(my_dict.has(4))       # Prints false
 				[/gdscript]
 				[csharp]
-				// You have to use Contains() here as an alternative to GDScript's `in` operator.
-				if (new Godot.Collections.Dictionary{{"godot", "engine"}}.Contains("godot"))
+				var myDict = new Godot.Collections.Dictionary
 				{
-				    // I am executed.
+				    {"Godot", 4},
+				    {210, null},
 				}
+				
+				GD.Print(myDict.Contains("Godot")) # Prints true
+				GD.Print(myDict.Contains(210))     # Prints true
+				GD.Print(myDict.Contains(4))       # Prints false
 				[/csharp]
 				[/codeblocks]
-				This method (like the [code]in[/code] operator) will evaluate to [code]true[/code] as long as the key exists, even if the associated value is [code]null[/code].
+				In GDScript, this is equivalent to the [code]in[/code] operator:
+				[codeblock]
+				if "Godot" in {"Godot": 4}:
+				    print("The key is here!") # Will be printed.
+				[/codeblock]
+				[b]Note:[/b] This method returns [code]true[/code] as long as the [param key] exists, even if its corresponding value is [code]null[/code].
```

Comment:
This small note may or may not be mergeable with the very first sentence. Something along the lines of _"Returns [code]true[/code] if the dictionary contains an entry with the given [param key], regardless of its corresponding value"_